sdk/transcation.py

```diff
@@ -0,0 +1,36 @@
+import action
+import json
+from pyevt import ecc, abi
+
+class Transcation:
+    def __init__(self, ChainId, BlockId, **kwargs):
+        self.chain_id = ChainId
+        self.block_id = BlockId
+        self.kwargs = kwargs
+        self.actions = []
+        self.transcation_extensions = []
+
+    def addAction(self, _action):
+        self.actions.append(_action.dict())
+    
+    def dumps(self):
+        ret = {
+            **self.kwargs,
+            "ref_block_num": self.block_id.ref_block_num(),
+            "ref_block_prefix": self.block_id.ref_block_prefix(),
+            "actions": self.actions,
+            "transcation_extensions": self.transcation_extensions
+        }
+        return json.dumps(ret)
+
+
+
+def get_sign_transcation(priv_keys, transcation):
```

Comment:
trx also has `ref_block_id` and `ref_block_prefix` and `expired` fields, for the first two, check the latest commits for pyevt